mypy/build.py

```diff
@@ -294,7 +298,9 @@ def default_lib_path(data_dir: str,
     # Add fallback path that can be used if we have a broken installation.
     if sys.platform != 'win32':
         path.append('/usr/local/lib/mypy')
-
+    assert path, "Could not resolve typeshed subdirectories. If you are using MyPy " \
```

Comment:
Im not sure we support 3.2... only 3.3+ are listed in `setup.py`, and the README states that we support 3.3+. We are planning on dropping 3.3 when it is EOL'd in a month I recall. And I agree it should be a constant.

As to the blocking error message, Im not sure. In general if we are in a state where things are "wrong" eg if there is a key error looking up builtins, that is an exception. I can see how the UX may overrule this. Perhaps raise using report_internal_error in mypy.errors?